ui/src/components/yaml/index.tsx

```diff
@@ -45,6 +45,7 @@ const Yaml = (props: IProps) => {
   const yamlRef = useRef<LegacyRef<HTMLDivElement> | undefined>()
   const diagnosisContentRef = useRef<HTMLDivElement>(null)
   const interpretEndRef = useRef<HTMLDivElement>(null)
+  const contentRef = useRef<HTMLDivElement>(null)
   const observerRef = useRef<MutationObserver | null>(null)
   const { data } = props
```

Comment:
This reference 'contentRef' appears to be unused and may be a dead reference, which can cause performance issues.